frontend/src/hooks/reportMapper/reportUIDataStructure.ts

```diff
@@ -5,7 +5,7 @@ export interface ReportDataWithTwoColumns {
 }
 
 export interface ValueWithUnits {
-  value: number
+  value: number | string
```

Comment:
keep the type only 1 type, recommend `number`

Reply:
Our team had a session to discuss this yesterday. “number | string” is  a must. The data from backend is a Number, but some of the data need to be calculated before display to component, and if we want two keep it in two decimal, it must be a string (e. 0.70).
The way that keeping the value in Number and calculate it in component is not working, since different row need different type, some need int, and others need float.

Reply:
> The way that keeping the value in Number and calculate it in component is not working, since different row need different type, some need int, and others need float.

Need a talk when code diff